build-scripts/config_common.cmake

```diff
@@ -86,9 +86,11 @@ endif ()
 
 if (WAMR_BUILD_JIT EQUAL 1)
   if (WAMR_BUILD_AOT EQUAL 1)
+    add_definitions("-DWASM_ENABLE_JIT=1")
```

Comment:
Just a personal opinion, it is better to operate `WAMR_BUILD_XXX` variables only in this block. And define `WASM_ENABLE_XXX` from L140.